Gems/LyShine/Code/Editor/ViewportIcon.cpp

```diff
@@ -49,12 +49,11 @@ void ViewportIcon::DrawImageAligned(Draw2dHelper& draw2d, AZ::Vector2& pivot, fl
         opacity);
 }
 
-void ViewportIcon::DrawImageTiled(Draw2dHelper& draw2d, IDraw2d::VertexPosColUV* verts, [[maybe_unused]] float opacity)
```

Comment:
Lyshine doesn't need the ability to render a transparent image anymore?

Reply:
This is for the icons that get displayed on the UI Editor viewport. The opacity parameter wasn't actually being used so I removed it. I believe one can set the opacity in the vert data that gets passed in.